src/state/QuestPlayStreakSyncState.ts

```diff
@@ -114,11 +120,9 @@ class QuestPlayStreakSyncState {
         const minimumRequiredPlayTimeInSeconds =
           questMeta?.eligibility?.play_streak.minimum_session_time_in_seconds
         if (
-          minimumRequiredPlayTimeInSeconds &&
-          currentPlayTimeInSeconds &&
+          minimumRequiredPlayTimeInSeconds !== undefined &&
```

Comment:
can we add `minimumRequiredPlayTimeInSeconds !== undefined && minimumRequiredPlayTimeInSeconds !== null`?

I really wish there was a non verbal way to check truthy values when value can be zero.

Reply:
sounds good and there's nothing wrong with being verbose 👌